registration-system/admin/pages_mail.php

```diff
@@ -154,7 +157,12 @@ private function buildQueryWhere() {
                 $where['abtyp'] = $_REQUEST['val_abtyp'];
             }
             if (isset($_REQUEST['check_nights'])) {
-                // TODO
+                $nights = $_REQUEST['val_nights'];
+                $conditions = [];
+                foreach ($nights as $night)
+                    $conditions['AND #'.$night] = ['anday[<=]' => $night, 'abday[>]' => $night];
```

Comment:
Definitionssache, aber ich finde, dass die Logik falsch ist.
Wenn ich wähle, dass ich an Personen mailen will die an Tag 1, 2, und 3 da sind, dann meine ich nicht 'und' an allen Tagen, sondern meine die Leute, die an min einem der gwählten Tage da sind (glaube ich). Trippy.

Reply:
ah. gute frage - ja, hier ist angenommen: an mindestens einer der selektierten nächte da. nicht gesagt 'NUR' in den nächten da.

Reply:
nutzt `fahrt->getPossibleDates()` in der neuen version